source/partials/release_notes/_release-20-2-0.md.erb

```diff
@@ -0,0 +1,84 @@
+<h4>Support for Rules on Config Repositories (Pipeline as Code)</h4>
+
+In order to provide fine-grained access control over the entities that are created by a config repository, GoCD adds support for defining rules on config repositories that specifies which environments, pipeline groups, pipelines the repository can refer to.
+Please refer to the [Specifying rules on Config Repository](https://docs.gocd.org/20.2.0/advanced_usage/pipelines_as_code.html#specifying-rules) for more information.
+
+
+<h4>Removal of Default SSL Support from GoCD</h4>
+
+Starting with GoCD 20.2.0, GoCD will no longer support configuring TLS. See [this Github Issue](https://github.com/gocd/gocd/pull/7669#issuecomment-580290432) for more information.
+
+
+<h4>Other Improvements</h4>
+
+* Starting this release, Alpine 3.11 and CentOS 8 based docker images for GoCD Server are <%= link_to 'available', 'https://hub.docker.com/search?q=gocd%2Fgocd-server&type=image' %>.
+* <%= link_to_issue 6031, 'Improve Pipeline Material auto update mismatch error' %>
+* <%= link_to_issue 7736, 'Improvements to wrapper properties examples' %>
+* <%= link_to_issue 7748, 'Improve Logging - Add plugin load/update/remove logs' %>
```

Comment:
Should this be a part of release notes? Logging was added to identify an issue.

Reply:
This was an in-general enhancement to the logging, hence considered as an enhancement.

@maheshp @arvindsv - Thoughts?

Reply:
Even though it was general, since it is a small change, I don't mind not calling it out separately.